install_config/configuring_local.adoc

```diff
@@ -0,0 +1,180 @@
+----
+
+[[local-volume-configure-local-provisioner]]
+=== Configure Local Provisioner
+OpenShift depends on an external provisioner to create persistent volumes for local devices and to clean them up when they are not needed (to enable reuse).
+
+[NOTE]
+====
+. The local volume provisioner is different from most provisioners and does not support dynamic provisioning.
+. Local volume provisioner requires that the administrators preconfigure the local volumes on each node and mount them under discovery directories. The provisioner then manages the volumes by creating and cleaning up PersistentVolumes for each volume.
+====
+
+
+This external provisioner needs to be configured by using a `ConfigMap` to relate directores with StorageClasses. This configuration must be created before the provisioner is deployed.
+
+[NOTE]
+====
+_(Optional)_ Create a standalone namespace for local volume provisioner and its configuration, for example:
+`oc new-project local-storage`
+====
+
+[source, yaml]
+----
+kind: ConfigMap
+metadata:
+  name: local-volume-config
+data:
+    "local-ssd": | <1>
+      {
+        "hostDir": "/mnt/local-storage/ssd", <2>
+        "mountDir": "/mnt/local-storage/ssd" <3>
+      }
+    "local-hdd": |
+      {
+        "hostDir": "/mnt/local-storage/hdd",
+        "mountDir": "/mnt/local-storage/hdd"
+      }
+----
+<1> Name of the StorageClass.
+<2> Path to the directory on the host. It must be a subdirectory of *_/mnt/local-storage_*.
+<3> Path to the directory in the provisioner pod. We recommend using the same directory structure as used on the host.
+
+With this configuration the provisioner creates:
+
+* One PV with StorageClass `local-ssd` for every subdirectory in *_/mnt/local-storage/ssd_*.
+* One PV with StorageClass `local-hdd` for every subdirectory in *_/mnt/local-storage/hdd_*.
+
+[[local-volume-deployment-local-provisioner]]
+=== Deploy Local Provisioner
+
+[NOTE]
+====
+Before starting the provisioner, mount all local devices and create a ConfigMap
+with storage classes and their directories.
+====
+
+Install the local provisioner from the link:https://raw.githubusercontent.com/jsafrane/origin/local-storage/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml[*_local-storage-provisioner-template.yaml_*] file.
+
+. Create a service account that allows running pods as a root user and use HostPath volumes:
++
+[source, bash]
+----
+$ oc create serviceaccount local-storage-admin
+$ oc adm policy add-scc-to-user hostmount-anyuid -z local-storage-admin
+----
+Root privileges are required for the provisioner pod for allowing it to delete
+content on local volumes. HostPath is required to access the
+*_/mnt/local-storage_* path on the host.
+
+. Install the template:
++
+[source, bash]
+----
+$ oc create -f https://raw.githubusercontent.com/jsafrane/origin/local-storage/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml
```

Comment:
I guess that `jsafrane` in the URL is fine for now, but it should be replaced with `openshift` once https://github.com/openshift/origin/pull/16538 is merged